### ISSUE TITLE
Travis CI doesn't run unit tests

### ISSUE THREAD

**Issue by gkossakowski** [sbt/sbt#2357]
The relevant fragment of travis script looks like this:

```
script:
  - sbt -J-XX:ReservedCodeCacheSize=128m "$SCRIPTED_TEST"
```

I believe we should add `sbt test` in front of running scripted tests.

**Comment by eed3si9n** [comment]
I should probably rename that variable to `$SBT_COMMAND` or something.

https://github.com/sbt/sbt/blob/d06b49d66dc3f594e8b385b0e2a020a49c62e059/.travis.yml#L18-L19

``` yaml
    - SCRIPTED_TEST="safeUnitTests"
    - SCRIPTED_TEST="otherUnitTests"
```

**Comment by gkossakowski** [comment]
Oh yes, I see where I got confused. Let me see what unit tests are being ran.

**Comment by gkossakowski** [comment]
Ok, so it runs most of units tests but not from `compileInterfaceProj`. Is that a simple omission in `safeProjects`?

``` scala
lazy val safeProjects: ScopeFilter = ScopeFilter(
  inProjects(mainSettingsProj, mainProj, ivyProj, completeProj,
    actionsProj, classpathProj, collectionProj, compileIncrementalProj,
    logProj, runProj, stdTaskProj, compilerProj),
  inConfigurations(Test)
)
```

**Comment by eed3si9n** [comment]
I'm guessing it is an omission.

**Comment by gkossakowski** [comment]
PR coming.

**Comment by eed3si9n** [comment]
o wow. It's actually failing too:

```
[info] DependencySpecification
[info] x Extracted source dependencies from public members
[error]  Set('E, 'A, 'G, 'B, 'D) is not equal to Set('E, 'A, 'G, 'B, 'C, 'D) (DependencySpecification.scala:29)
[error] Expected: ...'B, '[C, ']D)
[error] Actual:   ...'B, '[]D)
```

**Comment by gkossakowski** [comment]
Yes, that failure is how I discovered that tests are not being ran.

**Comment by eed3si9n** [comment]
Fixed by #2358
